queries.sql

```diff
@@ -1,3 +1,23 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE 'mon%';
```

Comment:
- [ ] Let's change `mon%` to `%mon`. Just like '`%mon%`' looks for mon anywhere in the string.
like `mon%`  will look for mon at the beginning of the string, and like `%mon`, will look for mon at the end of the string which we require.